internal/api/v1/testers.go

```diff
@@ -45,9 +49,20 @@ func ListTesters(testerService services.TesterService) fiber.Handler {
 //	@Failure		400	{object}	problemdetail.ProblemDetail
 //	@Failure		500	{object}	problemdetail.ProblemDetail
 //	@Router			/v1/testers.query [get]
-func SearchTesters(services.TesterService) fiber.Handler {
-	return func(c *fiber.Ctx) error {
-		return problemdetail.NotImplemented(c, "failed to search Testers")
+func SearchTesters(testerService services.TesterService) fiber.Handler {
+	return func(q *fiber.Ctx) error {
+		projectIDParam := q.Query("project_id", "")
+		projectID, err := strconv.Atoi(projectIDParam)
+		if err != nil {
+			return problemdetail.BadRequest(q, "failed to process request project id")
+		}
+		testers, err := testerService.FindByProjectID(q.Context(), int64(projectID))
+		if err != nil {
+			if errors.Is(err, sql.ErrNoRows) {
+				return q.JSON([]schema.Tester{})
+			}
+		}
```

Comment:
Error handling is incomplete when there is another kind of error, log and return server error response.